resources/charts/bitcoincore/templates/pod.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
   annotations:
     init_peers: "{{ .Values.connect | len }}"
 spec:
+  restartPolicy: Never
```

Comment:
I _think_ the preferred approach is to always add these to values.yaml ? i.e., values.yaml contains all the things we expect the user to change , and provides the defaults. the actual resource .yaml files and .tpl files are then strictly used to _apply_ the user provided / default values to the template. Could be wrong on this so might want to get a second opinion from @m3dwards / the helm docs.

Reply:
Yes, just on looks, this appears out of place in this file, and looks like it would fit in Values.yml

Reply:
OK perhaps I have it right now in 8e8d7d1510e3464ea5ab00420f68208a1a68ee27

Reply:
nice, looks much better!